src/services/httpsRequestUtils.ts

```diff
@@ -0,0 +1,46 @@
+import https from 'node:https'
+
+export async function makeHttpsRequest(
+  options: https.RequestOptions,
+  postData: Record<string, unknown> | undefined = undefined
+) {
+  return new Promise((resolve, reject) => {
+    const req = https.request(options, (response) => {
+      console.log('STATUS: ' + response.statusCode)
+
+      if (
+        (!response && response['statusCode'] < 200) ||
```

Comment:
Can this ever return true?  We're checking the response object is falsey and then trying to get the value of statusCode from it.